Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.cpp

```diff
@@ -386,6 +422,17 @@ namespace AZ
             {
                 const AZ::EntityId entityId = m_entityComponentIdPair.GetEntityId();
                 m_configuration.m_modelAsset = modelAsset;
+                RecomputeAssetQuality();
+
+                if (const AZStd::unordered_set<AZ::Name>& modelTags = m_configuration.m_modelAsset->GetTags(); !modelTags.empty())
```

Comment:
consider using auto to make this shorter
```suggestion
                if (const auto& modelTags = m_configuration.m_modelAsset->GetTags(); !modelTags.empty())
```